Sources/SwiftBasicFormat/Trivia+FormatExtensions.swift

```diff
@@ -45,7 +45,7 @@ extension Trivia {
 
   /// Returns the indentation of the last trivia piece in this trivia that is
   /// not a whitespace.
```

Comment:
If we make this public, could you explain the `isOnNewline` parameter as well?

Something like: `isOnNewline` specifies if the character before this trivia is a newline character, i.e. if this trivia already starts on a new line.

Reply:
Added more documentation